pkg/vm/builtins/modulo_test.go

```diff
@@ -0,0 +1,51 @@
+package builtins
+
+import (
+	"testing"
+
+	"github.com/NethermindEth/cairo-vm-go/pkg/vm/memory"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+	"github.com/stretchr/testify/require"
+)
+
+func TestModuloBuiltin(t *testing.T) {
+	mod := &ModBuiltin{ratio: 128, wordBitLen: 1, batchSize: 96, modBuiltinType: Add}
+	segment := memory.EmptySegmentWithLength(9)
+	segment.WithBuiltinRunner(mod)
+
+	v0, _ := new(fp.Element).SetString("1")
+	v1, _ := new(fp.Element).SetString("2")
+	v2, _ := new(fp.Element).SetString("3")
+	v3, _ := new(fp.Element).SetString("4")
+	v4, _ := new(fp.Element).SetString("5")
+	v5, _ := new(fp.Element).SetString("9")
+	v6, _ := new(fp.Element).SetString("7")
+	v7, _ := new(fp.Element).SetString("8")
```

Comment:
Pls avoid using `SetString` if you could use `SetUint64` instead. Leave the string method for a big number (that doesn't fit in uint64)